libbeat/outputs/elasticsearch/client.go

```diff
@@ -251,6 +251,20 @@ func (client *Client) publishEvents(
 	return nil, nil
 }
 
+func eventMetaValue(event *beat.Event, key string) string {
```

Comment:
Since there's nothing in this function that's specific to the Elasticsearch output, how about making the following changes to it?
- move it to https://github.com/ycombinator/beats/blob/master/libbeat/beat/event.go,
- instead of a function that takes `event *beat.Event` as the first parameter, make it a method with `e *beat.Event` as the receiver, and
- for the implementation simply return the result of `e.GetValue("@metadata."+key)`.